__tests__/integration/melt-tokens.test.js

```diff
@@ -1,10 +1,16 @@
-import { transactionUtils, constants, network, scriptsUtils } from '@hathor/wallet-lib';
+import { transactionUtils, constants, network, scriptsUtils, ScriptData } from '@hathor/wallet-lib';
 import { TestUtils } from './utils/test-utils-integration';
 import { WALLET_CONSTANTS } from './configuration/test-constants';
 import { WalletHelper } from './utils/wallet-helper';
 
 describe('melt tokens', () => {
   let wallet1;
+  let totalMinted = 1000;
+  let initialHTR = 10;
+  let meltedAmount = 0;
+  let htrMelted = 0;
```

Comment:
I changed all instances of hardcoded values with these calculated values, so that if we change the initial state (add more minted tokens) all tests will work accordingly and we wont need to update all tests.